src/olympia/addons/models.py

```diff
@@ -447,6 +447,7 @@ def is_soft_deleteable(self):
     def delete(self, msg='', reason=''):
         # To avoid a circular import.
         from . import tasks
+        from datetime import datetime as date
```

Comment:
Please add this import up top (in the first import block of the file, after `from operator import...`) instead. Also, `from datetime import datetime` should be fine (this is what we do everywhere else), no need to change the name.